process_pr.py

```diff
@@ -374,14 +377,25 @@ def modify_comment(comment, match, replace, dryRun):
     return 0
 
 
+def get_comment_emojis_(comment):
+    try:
+        return comment.reactions
+    except AttributeError:
+        import itertools
+
+        return defaultdict(
+            lambda: 0, zip((x.content for x in comment.get_reactions()), itertools.repeat(1))
+        )
+
+
 def set_comment_emoji_cache(dryRun, bot_cache, comment, repository, emoji="+1", reset_other=True):
     if dryRun:
         return
     comment_id = str(comment.id)
     if (
         (not comment_id in bot_cache["emoji"])
         or (bot_cache["emoji"][comment_id] != emoji)
-        or (comment.reactions[emoji] == 0)
+        or (get_comment_emojis_(comment)[emoji] == 0)
```

Comment:
why this was needed? I think `comment.reactions` should work for PyGithub deployed under /cvmfs/cms-ib.cern.ch  and cmssdt.

Reply:
https://github.com/PyGithub/PyGithub/pull/2813 merged upstream supports `comment.reactions` too

Reply:
My local version (which is probably newer than the one on cvmfs) doesn't have this property, instead it has `get_reactions` method returning list of reactions. I can change my local setup to use version from cvmfs and remove this wrapper.

Reply:
Latest release (2.1.1) [doesn't have](https://github.com/PyGithub/PyGithub/blob/v2.1.1/github/IssueComment.py#L124) `reactions` as property, but master [has](https://github.com/PyGithub/PyGithub/blob/main/github/IssueComment.py#L119). I will use version from cvmfs now.